README.md

```diff
@@ -133,7 +133,8 @@ Feel free to check the [issues page](../../issues/).
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse Success Team..
+I would like to thank Microverse Success Team.. 
+I would also like to thank Matee Safi for his contribution.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] `required`
- [ ] Good job with the template 👏 
- [ ] However there is a number of things missing please follow this [template](https://github.com/microverseinc/readme-template) and add the missing pieces 🥂

Reply:
Hello @mwanawabangona,
Thank you for making time to review my codes. 
However, I would appreciate it if you could mention what's missing in the Readme for me to fix because no reviewer has committed to the Readme after my last fix. I honestly don't know what you want me to do on the Readme. Please make it clear and easy for me to implement what's needed. Thank you.

Reply:
@Cosmosha 
Thank you for reaching out to me for clarity.

I have just noticed you have stacked everything in the `client`. a user doesn't have to click on client to see the license and your social handles etc. please make everything individual like the [template](https://github.com/microverseinc/readme-template) please compare your readme file and the template you have to follow. you will notice that yours is cut short.
If you have any more questions. will be more than happy to answer 😄 
I have  taken a screenshot of how your `readme file` is looking you can see no is able to see anything after client.

`screenshot of Your readme file`
![image](https://user-images.githubusercontent.com/66230306/216710234-88603606-3eeb-4dd7-bdfa-3d38f3378ee3.png)

Reply:
Hello @mwanawabangona 
Thank you so much for your feedback, I will look into it and get it fixed.

Reply:
Hello @mwanawabangona,
Concerning the background blur filter effect, I used css backdrop filter property to do that but I guess it doesn't support all browsers.
Please can you suggest something for me in place of the backdrop filter that could support all browsers?